content/blog/2019/11/2019-11-21-jenkins-health-advisor-by-cloudbees.adoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Jenkins Health Advisor by CloudBees is out!"
+tags:
+- jenkins
+- health
+- healthcheck
+- stability
+- 
+author: aheritier
+description: "Keep your Jenkins instance healthy with Jenkins Health Advisor by CloudBees"
+opengraph:
+  image: /images/post-images/jenkins-health-advisor/logo.png
```

Comment:
Would it be possible to use the full product name here and in other images?

Reply:
done